_activities/gsoc.md

```diff
@@ -49,7 +49,10 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 ### Student pages
 
-Student blogs are mandatory for CERN-HSF starting with 2022 GSoC edition! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/2022/blogs.html).
+The CERN-HSF 2022 student blogs are now available. [Check them out!]({{ site.baseurl }}/gsoc/2022/blogs.html)
+
+Student blogs are mandatory for CERN-HSF starting with 2022 GSoC edition! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/blogs/2022/blog_HSF_TheAdmins.html).
```

Comment:
maybe we should be more explicit and say that they are mandatory for positive evaluation.